### PR TITLE
fix(statusApp): resolving qml errors/warnings pt2

### LINKED ISSUE #8864: [StatusApp] Clean up console warnings Pt2
### Description

Clean up as much as ui related console warnings/errors can be cleaned in 8 hours (task to be continued through iterations)

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #8864

### What does the PR do
status app resolving QML errors/warnings

### Affected areas
status app

### CODE REVIEW

File: ui/app/AppLayouts/Browser/BrowserLayout.qml [hunk 1]
```diff
@@ -274,7 +274,7 @@ StatusSectionLayout {
             anchors.left: parent.left
             anchors.right: parent.right
             z: 50
-            tabComponent: webEngineView
+            tabComponent: !!webEngineView ? webEngineView : null
```
Comment (alexjba) [inline ui/app/AppLayouts/Browser/BrowserLayout.qml:277]: This component is defined below and should always be available.. When is this happening?

File: ui/app/AppLayouts/Profile/views/AppearanceView.qml [hunk 2]
```diff
@@ -39,7 +39,7 @@ SettingsContentBase {
 
     Item {
         id: appearanceContainer
-        anchors.left: parent.left
+        anchors.left: !!parent ? parent.left : undefined
```
Comment (alexjba) [inline ui/app/AppLayouts/Profile/views/AppearanceView.qml:42]: missing parent? In which case is this happening?

File: ui/app/AppLayouts/Profile/views/DevicesView.qml [hunk 3]
```diff
@@ -23,7 +23,7 @@ SettingsContentBase {
 
     Item {
         width: root.contentWidth
-        height: parent.height
+        height: !!parent ? parent.height : 0
```
Comment (alexjba) [inline ui/app/AppLayouts/Profile/views/DevicesView.qml:26]: missing parent? In which case is this happening?

**Review by alexjba — COMMENTED** [review]
Looks good in general. Just a few questions because I don't understand the context in which some of the warnings are prompted.

File: ui/app/AppLayouts/Profile/views/AppearanceView.qml (hunk 2 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Profile/views/AppearanceView.qml:42]: that's very usual in StackLayout, in the moment when layouts are switched

File: ui/app/AppLayouts/Profile/views/DevicesView.qml (hunk 3 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Profile/views/DevicesView.qml:26]: same as above

File: ui/app/AppLayouts/Browser/BrowserLayout.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Browser/BrowserLayout.qml:277]: when the browser view is loading.

**Review by caybro — APPROVED** [review]
LGTM imo

**Review by alexjba — APPROVED** [review]
Thank you @alexandraB99. LGTM 👍

**Review by noeliaSD — APPROVED** [review]
LGTM! Thanks for doing that!! :)

**Pull request merged**